### PR TITLE
new clang-format

### PR DESCRIPTION
**Pull request by edefilippo**
Describe your proposal.

Mention any issue this PR is resolved or is related to.
clang-format : new light corrections
---

Checklist:

* [ ] Rebased against `dev` branch
* [ ] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [ ] Followed [the seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

**Comment by jose-luis-rs** [comment]
Hello Enrico, have you run the code to simulate some events?

File: asydata/chimeraData/R3BAsyChimeraHitData.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Double_t GetToF() const { return fToF; }
+
+    /** Modifiers **/
+    void SetDetID(Int_t ndet) { fDetId = ndet; };
+    void SetELoss(Double_t eloss) { fELoss = eloss; };
+    void SetFast(Double_t fast) { fFast = fast; };
+    void SetSlow(Double_t slow) { fSlow = slow; };
+    void SetTof(Double_t t) { fToF = t; };
+
+  protected:
```
Comment (jose-luis-rs) [inline asydata/chimeraData/R3BAsyChimeraHitData.h:38]: private:

File: asydata/chimeraData/R3BAsyChimeraPoint.h [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    /** Modifiers **/
+    void SetPositionOut(TVector3 pos);
+    void SetMomentumOut(TVector3 mom);
+    void SetDetCopyID(Int_t id) { fDetCopyID = id; };
+
+    /** Output to screen **/
+    //virtual void Print(const Option_t* opt) const;
+
+  protected:
```
Comment (jose-luis-rs) [inline asydata/chimeraData/R3BAsyChimeraPoint.h:94]: private:

File: chimera/pars/R3BAsyChimeraCalPar.h [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+// ---------------------------------------------------------------
+// -----             R3BTwimCalPar source file               -----
+// -----       Created 24/11/19 by J.L. Rodriguez-Sanchez    -----
```
Comment (jose-luis-rs) [inline chimera/pars/R3BAsyChimeraCalPar.h:16]: Please, update date and author

File: chimera/pars/R3BAsyChimeraContFact.cxx [hunk 4]
```diff
@@ -0,0 +1,77 @@
+// ----------------------------------------------------------------------
+//								    -----
+//  Factory for the parameter containers in libR3BAsyChimera           -----
+//         Created 03/11/19  by J.L. Rodriguez-Sanchez              -----
```
Comment (jose-luis-rs) [inline chimera/pars/R3BAsyChimeraContFact.cxx:4]: Please, update date and author

File: chimera/sim/R3BAsyChimera.cxx [hunk 5]
```diff
@@ -0,0 +1,242 @@
+// -------------------------------------------------------------------------
+// -----                        R3BAsyChimera source file                  -----
+// -----                  Created 24/11/17  by H.Alvarez-Pol          -----
```
Comment (jose-luis-rs) [inline chimera/sim/R3BAsyChimera.cxx:3]: Please, update date and author

**Comment by jose-luis-rs** [comment]
Hello Enrico

Please, add the main asyeos repository to your local asyeos folder with the command:

        git remote add mainrepo [git@github.com](mailto:git@github.com):R3BRootGroup/asyeos.git
        git fetch --all

and then rebase the main repository to synchronize both with the command:

        git rebase mainrepo/dev

After doing this, you could apply the clang-format to your new files by using the command:

        source util/clang-format-all.sh

Then you will need to add again the modified files with the commands:

        git add "file name 1" "file name 2" "file name 3" ...
        git commit -m 'Clang-format'
        git push

And finally you should make a squash of your commits (just to have one main comment in your pull request):

        git rebase -i mainrepo/dev

This will open an editor where you can select your main commits (pick) and the secondary ones (squash or s)
For more details, visit:
https://github.com/AnarManafov/GitWorkflow/blob/master/GitWorkflow.markdown

Let me know in case of problems with this last step.

File: chimera/pars/R3BAsyChimeraContFact.cxx (hunk 4 above)
Comment (edefilippo) [reply chimera/pars/R3BAsyChimeraContFact.cxx]: Done. We have preferred to update authors and date in .h (header file)    
keeping 
// -------------------------------------------------------------------------
// -----                        R3BAsyxxxxx source file            -----
// -------------------------------------------------------------------------
in cxx source files as seen for other R3Broot devices

**Comment by edefilippo** [comment]
After i add mainrepo, the following command gives error

fetch --all

Fetching origin
Fetching mainrepo
***@***.***: Permission denied (publickey).
fatal: Could not read from remote repository.

I tried to add mainrepo  with my gpg valid token (in place of initial 
***@***.***) and also with your line below

|git remote add mainrepo ***@***.***:R3BRootGroup/asyeos.git |

|but in both cases |||i obtain the same error when i do fetch. Fetch 
from origin is ok.

Enrico

||

**Comment by jose-luis-rs** [comment]
The copy&paste failed, I should be:
>git remote add mainrepo git@github.com:R3BRootGroup/asyeos.git
>git fetch --all

To remove the previous link to the main repository you can use the command:
git remote rm mainrepo

Sorry!

**Pull request closed without merging**